src/valkey-benchmark.c

```diff
@@ -1958,6 +2007,51 @@ int main(int argc, char **argv) {
             free(cmd);
         }
 
+        if (test_is_selected("function_load")) {
+            char *script = generateFunctionScript(config.num_functions, 0);
+            len = redisFormatCommand(&cmd, "function load replace %s", script);
+            benchmark("FUNCTION LOAD", cmd, len);
```

Comment:
Load the same script many times? We could optimize this case, but that would make the benchmark useless. :laughing: